cmd/mistry/main.go

```diff
@@ -248,30 +249,37 @@ EXAMPLES:
 					return err
 				}
 
+				if !jsonResult {
+					fmt.Println("Logs can be found at", baseUrl+"/"+bi.URL)
+				}
+
 				if verbose {
 					fmt.Printf(
 						"\nResult:\nStarted at: %s ExitCode: %v Params: %s Cached: %v Coalesced: %v\n\nLogs:\n%s\n",
 						bi.StartedAt, bi.ExitCode, bi.Params, bi.Cached, bi.Coalesced, bi.Log)
 				}
 
 				if jsonResult {
-					fmt.Printf("%s", body)
+					fmt.Printf("%s\n", body)
 				}
 
 				if bi.ExitCode != 0 {
+					if bi.ErrLog != "" {
+						fmt.Fprintln(os.Stderr, "Container error logs:\n", bi.ErrLog)
+					}
 					return fmt.Errorf("Build failed with exit code %d", bi.ExitCode)
 				}
 
 				if verbose {
-					fmt.Printf("Copying artifacts to %s", target)
+					fmt.Println("Copying artifacts to", target)
```

Comment:
Append a "..." here so that it's apparent that the operation might take some time.